examples/amp-story/interactive_size.html

```diff
@@ -0,0 +1,319 @@
+<!-- 
```

Comment:
Please remove as it's unrelated to this PR

Reply:
Oops idk how it got here. Removed